binaries/localserver/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"github.com/scootdev/scoot/local/server"
+	"log"
+)
+
+// A Local Scoot server.
+func main() {
+	scootdir, err := protocol.Locate()
+	if err != nil {
+		log.Fatal("Error locating Scoot instance: ", err)
```

Comment:
Yeah, log.Fatal logs and then calls os.Exit.

https://golang.org/pkg/log/#Fatal